gcloud/exceptions.py

```diff
@@ -171,6 +172,9 @@ def make_exception(response, content, use_json=True):
     :rtype: instance of :class:`GCloudError`, or a concrete subclass.
     :returns: Exception specific to the error response.
     """
+    if six.PY3 and isinstance(content, bytes):
+        content = content.decode('utf-8')  # pragma: NO COVER Py3K
```

Comment:
To avoid modifying how the code worked under Python 2 and thereby possibly introducing more problems.  If you prefer I'll remove the conditional.